README.md

```diff
@@ -0,0 +1,175 @@
+# Open Context for Node.js
+
+This is a simple library for providing Distributed Context Problem
+in Node.js programs.
+
+DCP is most beneficial in large service oriented architectures. Each request
+has some associated metadata, and we want to make sure that data stays with
+the tree of RPCs that service the root request.
+
+The `Context` object in this library stores that request metadata, or baggage.
+It's intended to be implemented into RPC libraries, who would be responsible
+for actually serializing the data onto the wire.
+
+It also supports 'joining', a process where we copy all values from one
+context to another.
```

Comment:
OpenCtx supports "joining". Both requests and responses can carry baggage. Joining response context with the context in hand carries the baggage forward through subsequent requests and responses. Joining parallel responses may involve property specific logic, like taking the lesser of two deadlines, merging sets of receipts, or computing the latter of logical clocks.